nengo_bones/tests/test_pr_number.py

```diff
@@ -0,0 +1,56 @@
+# pylint: disable=missing-docstring
+
+import os
+
+from click.testing import CliRunner
+import pytest
+
+from nengo_bones.scripts import pr_number
+from nengo_bones.tests.utils import assert_exit, make_has_line, write_file
+
+
+def test_pr_number_repo(monkeypatch):
+    # monkeypatch so we don't rely on Github connection
+    monkeypatch.setattr(pr_number, "get_issue_count", lambda repo: 9)
```

Comment:
I think it's better to monkeypatch `requests.get` so that we limit the amount of stuff that we're mocking out. We can also then test that we're requesting from the correct URL and that we correctly handle the JSON payload that we get back. It makes it more obvious what we have to change if Github changes how they do things, but still doesn't require Github to be up and running for us to do our tests.

Reply:
We also wouldn't have to do a `pragma: no-cover`